### ISSUE TITLE
version confusion?

### ISSUE THREAD

**Issue by jubalh** [lxde/obconf-qt#21]
Last version was 0.1.0.
https://github.com/lxde/obconf-qt/commit/e9df1be7e256470670bdd1f2e7af0265b20bd5a1 bumps the version 0.2.0.
However on https://downloads.lxqt.org/obconf-qt/ and https://github.com/lxde/obconf-qt/releases it seems you released as 0.11.0.

**Comment by agaida** [comment]
upps - it seems to be my fault back in 2015 with the 0.9 release - we had no release for obconf and numbered it as 0.9.0 - so i numbered it in the release scripts as 0.11.0. Anyways, it is in the wild now, so i guess we shoud set the package version to 0.11.0

**Comment by pmattern** [comment]
Background of the current versioning:
When the code was transferred from git.lxde.org to GitHub a downgrade of the commit numbering took place by accident. This was causing confusion in the Arch Linux AUR packages where the helper tools were erroneously considering packages outdated all the time.
So the decision was made to bump the upstream version. 0.9 was probably chosen expecting that the versioning of ObConf-Qt should be adjusted to the one of LXQt one day anyway.

But looking back I'd say choosing 0.9 was a mistake.
The development of ObConf-Qt is decoupled from LXQt the same way as e. g. ComptonConf or pavucontrol-Qt and the development itself sure didn't warrant an increase from 0.1 to 0.9 either.

What I'd personally suggest is put tags / releases 0.9 and 0.11 in the trash can, create a proper release 0.2.0 and communicate the whole mess on the mailing list.
This would be a bit awkward in particular for downstream but allow for introducing a reasonable versioning from now on. If on the other hand we refrain from this step now we'd be forced to stick with a versioning that basically doesn't make sense permanently.

**Comment by jubalh** [comment]
For distributions that have 0.9 in their repos this will be a mess, right?
If the README is right this is only the case for Arch and Debian. Though I don't know which version they use.

**Comment by paulolieuthier** [comment]
"Downgrading" to the correct version is not the way to go. We should continue from where it is now.

**Comment by pmattern** [comment]
As said above downgrading would be awkward, sure.

But the downstream situation is worth having a look: in Arch Linux it's in the AUR only and hence in no official repository, in Debian it's in testing and experimental (where a downgrade [took already place](https://packages.debian.org/search?keywords=obconf-qt&searchon=names&suite=all&section=all) due the problem we're discussing here), in Fedora and openSUSE it's lacking completely.

So I think downstream issues wouldn't really hurt in this particular case.

**Comment by paulolieuthier** [comment]
In that case it should be done really soon.

**Comment by pmattern** [comment]
We need to hurry up and take a decision quickly no matter what we decide.
Currently there's an inconsistency between tag and CMake which shouldn't get too wide-spread downstream either.

**Comment by agaida** [comment]
```
AUR                     <td>0.9.0-2</td>                     28                                                           http://aur.archlinux.org              <td>0.9.0-2</td>
AUR                     <td>0.1.0.18.gfe11967-1</td>         28                                                           http://aur.archlinux.org              <td>0.1.0.18.gfe11967-1</td>
openSUSE    obconf-qt                                                                                           http://software.opensuse.org/package/obconf-qt
Ubuntu      obconf-qt                              0.9.0+20150723-1   73K             all                       http://packages.ubuntu.com/wily/obconf-qt
Ubuntu      obconf-qt                              0.9.0+20151227-1   74K             all                       http://packages.ubuntu.com/xenial/obconf-qt
Ubuntu      obconf-qt                              0.9.0+20151227-2   74K             all                       http://packages.ubuntu.com/yakkety/obconf-qt
NetBSD      obconf-qt                              0.1.0              63K             wip                       http://pkgsrc.se/wip/obconf-qt
Debian      obconf-qt                              0.9.0+20151227-2   74K             all                       http://packages.debian.org/stretch/obconf-qt
Debian      obconf-qt                              0.9.0+20151227-2+b 74K             all                       http://packages.debian.org/sid/obconf-qt
Debian      obconf-qt                              0.10.96~25-gbc999b 49K             all                       http://packages.debian.org/experimental/obconf-qt
Debian      obconf-qt-dbgsym                       0.9.0+20151227-2+b                 all                       http://packages.debian.org/sid/obconf-qt-dbgsym
Debian      obconf-qt-dbgsym                       0.10.96~25-gbc999b                 all                       http://packages.debian.org/experimental/obconf-qt-dbgsym
Debian      obconf-qt-l10n                         0.10.95~20-g4a053f 33K             all                       http://packages.debian.org/experimental/obconf-qt-l10n
Gentoo      obconf-qt                              0.9.0_p20150729  
```
so 0.9.0 and 0.9.0+2015... are in the wild for some days - in 3 official ubuntu repos (that means in all derivative repos too - and far more important - it is installed with this number), in debian testing  and unstable (same as with ubuntu for derivatives), Gentoo - hey, a number is a number is a number. and in debian experimental is 0.10.96 - as i followed the KDE numbering scheme, the beta before 0.11.0. Ok, for debian i could use a special numbering with an epoch [1:]0.2.0 - but this is not worth the effort.

**Comment by jubalh** [comment]
> in Fedora and openSUSE it's lacking completely.
Not true. 0.11.0 lives in [X11:LXQt](https://build.opensuse.org/package/show/X11:LXQt/obconf-qt) devel repo. And before that 0.9.0 lived in X11:lxde for years. But it wasn't in the official distribution so only users who enabled this repo can have it.

**Comment by agaida** [comment]
> upstream_url: https://github.com/lxde/obconf-qt/archive/0.9.0.tar.gz
> upstream_version: 0.9.0
> version: 0.9.0+20151227-2
so i think we should only fix the version in cmake to 0.11.0 and be done with - and after that "fix" solve some real problems :D

**Comment by jubalh** [comment]
If we decide to bump the version up then probably to 0.11.1 even, since we already released 0.11.0 tarballs with different content? There is a chance people already grabbed that one.

**Comment by agaida** [comment]
To be true - i would do nothing right now - obconf-qt is released as 0.11.0, there is no need for anyone to use the cmake packaging thing. We should fix it to prevent further confusion later. But a number in a script that nobody use in imho not a reason for a point release. This would be a bit overkill.

And if we really want to fix it we should provide proper versions in the CMakelists.txt - beside such things like minimum Qt version. I wrote a bug for Qt already.
